build/src/components/05-gallery/gallery-ads.js

```diff
@@ -0,0 +1,92 @@
+import log from 'loglevel';
+
+/**
+ * Ad handler for the gallery.
+ */
+class GalleryAds {
+
+  /**
+   * Initialize swiper instance.
+   *
+   * @param {Object} swiper
+   *   The swiper instance.
+   */
+  init(swiper) {
+    if (typeof(Drupal.ad_entity) === "undefined") {
+      return;
+    }
+
+    this.swiper = swiper;
+    this.initializeAllAds();
+  }
+
+  /**
+   * Get slides with an ad.
+   *
+   * @returns {Array}
+   *   Slides containing an ad.
+   */
+  getAdSlides() {
+    let ad_slides = [];
```

Comment:
Unless the variable is being reassigned it's value you should always use `const` instead of `let`